### ISSUE TITLE
WorldEdit on Fabric server does not allow client to connect

### ISSUE THREAD

**Issue by Sushant-Padha** [EngineHub/WorldEdit#1935]
### WorldEdit Version

7.2.7

### Platform Version

Fabric 0.11.6

### Confirmations

- [X] I am using the most recent Minecraft release.
- [X] I am using a version of WorldEdit compatible with my Minecraft version.
- [X] I am using the latest or recommended version of my platform software.
- [X] I am NOT using a hybrid server, e.g. a server that combines Bukkit and Forge. Examples include Arclight, Mohist, and Cardboard.
- [X] I am NOT using a fork of WorldEdit, such as FastAsyncWorldEdit (FAWE) or AsyncWorldEdit (AWE)

### Bug Description

When WorldEdit is added into the `mods` folder of a fabric server, it does not allow the client to join it, with or without the client having the mod installed.
The client side displays "Failed to connect to server." as the error message.

### Expected Behavior

Client connects to server normally and WorldEdit are usable.

### Reproduction Steps

1. Run a Fabric 0.11.6 server, keeping only the "WorldEdit 7.2.7 for Fabric" mod in the `mods` folder.
3. Run Minecraft client with a Fabric 0.11.6 using Minecraft **1.17** (not 1.17.1) with only the "WorldEdit 7.2.7 for Fabric" mod installed.
4. Connect to the server.

### Anything Else?

As stated above, I am *NOT* using the latest version (1.17.1) but 1.17 instead. However, it is still the most recent major version, i.e., Minecraft 1.17.

Following is a compressed version of the server log:
```log
[16:35:21] [main/INFO]: Loading for game Minecraft 1.17
[16:35:22] [main/INFO]: [FabricLoader] Loading 9 mods:
	- fabric-api-base@0.3.0+a02b446313
	- fabric-command-api-v1@1.1.2+6cefd57713
	- fabric-events-interaction-v0@0.4.9+a722d8c013
	- fabric-lifecycle-events-v1@1.4.4+a02b446313
	- fabric-networking-api-v1@1.0.12+6cefd57713
	- fabricloader@0.11.6
	- java@16
	- minecraft@1.17
	- worldedit@7.2.7+9f3e795
[16:35:22] [main/INFO]: SpongePowered MIXIN Subsystem Version=0.8.2 Source=file:/D:/minecraft-servers/Survival-Lands/fabric-server-launch.jar Service=Knot/Fabric Env=SERVER
[16:35:22] [main/INFO]: Compatibility level set to JAVA_16
[16:35:30] [main/WARN]: @Inject(@At("INVOKE")) Shift.BY=3 on fabric-lifecycle-events-v1.mixins.json:server.WorldChunkMixin::handler$zzn000$onLoadBlockEntity exceeds the maximum allowed value: 0. Increase the value of maxShiftBy to suppress this warning.
[16:35:32] [main/INFO]: Got request to register class com.sk89q.worldedit.fabric.FabricPlatform with WorldEdit [com.sk89q.worldedit.extension.platform.PlatformManager@5521f423]
[16:35:32] [main/INFO]: WorldEdit for Fabric (version 7.2.7+9f3e795) is loaded
[16:35:34] [main/INFO]: Environment: authHost='https://authserver.mojang.com', accountsHost='https://api.mojang.com', sessionHost='https://sessionserver.mojang.com', servicesHost='https://api.minecraftservices.com', name='PROD'
[16:35:35] [main/INFO]: Registering commands with com.sk89q.worldedit.fabric.FabricPlatform
...
[16:37:14] [Server thread/INFO]: Time elapsed: 74871 ms
[16:37:14] [Server thread/INFO]: Done (91.544s)! For help, type "help"
[16:37:55] [Netty Server IO #2/WARN]: An exception was thrown by net.fabricmc.fabric.impl.networking.server.ServerLoginNetworkAddon$$Lambda$5322/0x00000008016dfb98.operationComplete()
java.lang.NoSuchMethodError: 'void net.minecraft.class_2535.method_10760(int, boolean)'
	at net.fabricmc.fabric.impl.networking.server.ServerLoginNetworkAddon.lambda$sendCompressionPacket$2(ServerLoginNetworkAddon.java:117) ~[81bd2f6b-b5c6-4244-8c98-8f3efc20393e.jar:?]
	at io.netty.util.concurrent.DefaultPromise.notifyListener0(DefaultPromise.java:511) ~[intermediary-server.jar:?]
	at io.netty.util.concurrent.DefaultPromise.notifyListenersNow(DefaultPromise.java:485) ~[intermediary-server.jar:?]
	at io.netty.util.concurrent.DefaultPromise.notifyListeners(DefaultPromise.java:424) ~[intermediary-server.jar:?]
	at io.netty.util.concurrent.DefaultPromise.addListener(DefaultPromise.java:162) ~[intermediary-server.jar:?]
	at io.netty.channel.DefaultChannelPromise.addListener(DefaultChannelPromise.java:95) ~[intermediary-server.jar:?]
	at io.netty.channel.DefaultChannelPromise.addListener(DefaultChannelPromise.java:30) ~[intermediary-server.jar:?]
	at net.minecraft.class_2535.method_36942(class_2535.java:207) ~[intermediary-server.jar:?]
	at net.minecraft.class_2535.method_10761(class_2535.java:197) ~[intermediary-server.jar:?]
	at io.netty.util.concurrent.AbstractEventExecutor.safeExecute(AbstractEventExecutor.java:163) [intermediary-server.jar:?]
	at io.netty.util.concurrent.SingleThreadEventExecutor.runAllTasks(SingleThreadEventExecutor.java:404) [intermediary-server.jar:?]
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:465) [intermediary-server.jar:?]
	at io.netty.util.concurrent.SingleThreadEventExecutor$5.run(SingleThreadEventExecutor.java:884) [intermediary-server.jar:?]
	at java.lang.Thread.run(Thread.java:831) [?:?]
[16:37:55] [Server thread/INFO]: com.mojang.authlib.GameProfile@79fa0335[id=<null>,name=havokwrecker,properties={},legacy=false] (/127.0.0.1:63750) lost connection: Disconnected
...
```

**Comment by me4502** [comment]
1.17 is not supported, we only support latest patch releases.

**Comment by Sushant-Padha** [comment]
Okay.
I will test with 1.17.1 now.

**Comment by Sushant-Padha** [comment]
Fixed in latest version (Minecraft 1.17.1).
